include/swift/AST/DiagnosticsSema.def

```diff
@@ -3773,6 +3773,11 @@ NOTE(missing_particular_case,none,
      "add missing case: '%0'", (StringRef))
 WARNING(redundant_particular_case,none,
         "case is already handled by previous patterns; consider removing it",())
+WARNING(redundant_particular_literal_case,none,
+        "literal value is already handled by previous pattern; "
+        "is this intentional?",())
```

Comment:
Nit: since there's no way to silence this warning without removing the redundant case, it's not actually possible for the user to answer the question "is this intentional?"--therefore, probably best to use the same wording as above (i.e. "consider removing it").